### ISSUE TITLE
A few issues I've noticed for the past 2 months of using this library.

### ISSUE THREAD

**Issue by rubby1337** [leocb/MaterialSkin#96]
- Changing colorschemes on runtime disables the button animation.
- Typing in MaterialTextBoxes generates empty space between the cursor and the end of the text: ![image](https://i.imgur.com/s273zbi.gif)
- When there are multiple checkboxes in one form/tab, the first one displays some type of an animation: 
![image](https://user-images.githubusercontent.com/66013187/83121241-b9dffb80-a0da-11ea-9560-8e9c9db3d021.png)
- Using an OpenFiledDialog/SaveFileDialog from a MaterialForm freezes the program. This appears very rarely and it can be prevented by enabling the ShowHelp property on the OpenFiledDialog/SaveFileDialog.

Sorry for my terrible english lol.

**Comment by leocb** [comment]
Hi, thanks for reporting!
Let's tackle those issues one by one:
First some generic questions:
- What version of material skin are you using?
- What version of the .NET framework is your project using, there're some known issues with version 4.7.2

Second for your report:
I'm using the MaterialSkin Example project from this repo

1. I Can't reproduce this button bug, could you share more details of when it happens?
2. This was a known issue a few versions back, it happens when the user does not have the Roboto font installed on the system, I tought i solved this by packing and loading the font into memory a while back, this could be a scaling issue, are you using DPI scaling on your system? (125%, 150%?) if so, does this persist when you change it back to 100%?
3. That's the expected behavior, it means the control has the focus, if you press spacebar it will check the box, press tab and the focus will change
4. Never had this issue, and "very rarely" will make it hard to debug, if you can find a setup that freezes the program everytime, then I'll try to fix it

Thanks again for reporting

**Comment by rubby1337** [comment]
Hi, I'm using .net framework 4.6.1 with materialskin 2.1.2 (the latest version from nuget).
Here's an example of the button hovering animation glitch:
![m91lJhxu78](https://user-images.githubusercontent.com/66013187/83150694-3d621280-a104-11ea-9396-877deffc05c3.gif)
And here's the code:
```c#
if (primaryColor.SelectedIndex == 0 || secondaryColor.SelectedIndex == 0)
           return;

Primary val1 = (Primary)Enum.Parse(typeof(Primary), primaryColor.SelectedItem.ToString());
Accent val2 = (Accent)Enum.Parse(typeof(Accent), secondaryColor.SelectedItem.ToString());

materialSkinManager.ColorScheme = materialSkinManager.Theme == MaterialSkinManager.Themes.DARK ?
materialSkinManager.ColorScheme = new ColorScheme(val1, val1, val1, val2, TextShade.WHITE) :
materialSkinManager.ColorScheme = new ColorScheme(val1, val1, val1, val2, TextShade.BLACK);
```
The code is kinda messy and bad but you get the point.

About the checkboxes, I thought that wasn't normal behavior. My bad.

Making the DPI scaling 100% fixed the textboxes problem, thanks!

It seems like the Open/Save file dialog issue is because of something with the .net framework, other people are experiencing it too. My bad for not googling before posting here.

**Comment by leocb** [comment]
no problem!, glad to know

For your button issue, this happens because you're setting the dark and light colors the same as your base color (you use `val1` for all three values), I use the darker/lighter versions to do those animations, since they're all the same color, you can't see it. Try setting them to different tones

as for the textBox: Glad to know DPI scaling mess those up, probably will be fixed when i do #93

**Comment by rubby1337** [comment]
Thanks for your response. I thought the dark/white color params are just base colors for each theme. Closing the issue.
